src/main/java/org/opentripplanner/openstreetmap/model/OSMWay.java

```diff
@@ -137,8 +137,23 @@ public boolean isBackwardEscalator() {
     return isEscalator() && "backward".equals(this.getTag("conveying"));
   }
 
+  /**
+   * Returns true if the way is considered an area.
+   *
+   * An area can be specified as such, or be one by default as an amenity.
+   */
   public boolean isArea() {
-    return isTag("area", "yes");
+    return (
```

Comment:
Since this is changing the logic, can you rename this method to `isRoutableArea`?

Reply:
I need to use another name as some of the areas are not always routable, like parking or room.

Reply:
I have renamed it has `isAreaWay` as I can't find another good name to use, even is the object is already a way. It can't be used on multipolygons.

Reply:
We use the "routable" convention to tell if it's considered routable _to OTP's graph builder_ so it's the correct term in our context.

We also have `isRoutable` which does something similar.

But you're right, in later stages we look at permissions and prune islands so it's not definitive routability. The Javadoc shoudl reflect that.

Reply:
`indoor=room` will return yes in this method but it is definitely not routable at all.

Reply:
So why do you include it if you don't want to route on them?